src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -279,6 +277,24 @@ public Optional<List<String>> getSiblings(String serverName) {
                 });
     }
 
+    /**
+     * Get the servers in the same group as the given server, as an optional list of {@link ServerInfo}
+     * <p>
+     * If the server is not in a group, use the fallback group
+     * If the fallback is disabled, return an empty optional
+     *
+     * @param serverName The server name
+     * @return The servers in the same group as the given server, empty if the server is not in a group and fallback is disabled
+     */
+    @NotNull
+    public List<RegisteredServer> getSiblingsServers(String serverName) {
```

Comment:
Should probably be "getSiblingServers" rather than "getSiblings"

Reply:
There is another method with the same name and paramenters.

Reply:
![image](https://github.com/WiIIiam278/Velocitab/assets/56083016/e380d883-d297-4624-9bfa-f2ec99e62b6f)
![image](https://github.com/WiIIiam278/Velocitab/assets/56083016/4df5aa64-5b75-454c-91c6-cf0171c9c966)
What about this change?

Reply:
Good ifea